.github/workflows/deploy-changed-cf.yaml

```diff
@@ -149,12 +153,15 @@ jobs:
     if: github.ref == 'refs/heads/master' || github.event_name == 'workflow_dispatch'
     outputs:
       deploy-rds: ${{ steps.set-name.outputs.deploy-rds && github.ref == 'refs/heads/master' }}
+    env:
+      CLUSTER_ENV: ${{ matrix.environment-type }}
```

Comment:
Can you move this code under environment (line 164)? Just to make it easier to see where `environment-type` comes from